Makefile

```
@@ -270,13 +278,23 @@ destroy-cumulus: cumulus-init
 force: ;
 
 # ---------------------------
+
+upload-cirrus-version:
```

`upload-cirrus-version` is not the name of a file produced by the recipe here so you should mark it as `PHONY`.

```suggestion
.PHONY: upload-cirrus-version
upload-cirrus-version:
```